azerothcore/azerothcore-wotlk
# refactor(Core): apply clang-tidy modernize-use-nullptr

- Part of https://github.com/azerothcore/azerothcore-wotlk/issues/3820
- Apply `clang-tidy` with `modernize-use-nullptr`
https://clang.llvm.org/extra/clang-tidy/checks/modernize-use-nullptr.html

### How to test the changes

This PR is only about refactoring, it is then not meant to change anything of the program behaviour.
To test this, it is sufficient to:

- (Not mandatory but would be nice to build with `-DTOOLS=1` [extract the client files](https://www.azerothcore.org/wiki/Extract-Client-Data) and use the newly extracted data to run the server)
- Run the server
- Access the game and play around a bit
- All good so far? Then this should be good to go

## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
